src/api/store/media_library.py

```diff
@@ -196,7 +204,9 @@ def get_most_recent(self, args, context: Context):
             else:
                 query |= qObj
 
+        count = 0
         if not upper_limit and not lower_limit:
+            count = Media.objects.filter(query).distinct().count()
             images = Media.objects.filter(query).distinct().order_by("-id")[:limit]
         else:
             images = (
```

Comment:
if you go this path, count=0.  Is that the intention?

Reply:
Yes @BradHN1, count is  0 every time after the first time we retrieve the actual count within the "if not upper and not lower_limit" block (Which essentially means every first time we run the query). I dont bother to recalculate the  count when the limit values are available because I know that by the time upper_limit and lower_limits are available, the user has definitely used the "load more" feature so it's the same query, nothing has changed and chances are the total count is the same (The frontend knows not to update the the total count value on 'load more' requests).

At the time I was saving us unnecessary database access, but now that I am explaining it to you, I think I can see how it is very possible that content would have changed either by (adding/deleting) by other admins at any given time, so its better to retrieve the updated count value as much as we can 🤣 🤣 🤣 🤣 🤣